documentation/internal/Home.md

```diff
@@ -7,5 +7,9 @@ Please see [ReadMe](../__README.md) for information about maintaining this docum
 - [Setup](setup/Setup.md)
 - [Overview of Navigation/URL structure](Overview-of-Navigation-&-URL-structure.md)
 - [Filters for Applications](Filters-for-Applications.md)
-- [Development workflow](Development-Workflow.md)
+
+### Development
+
+- [Workflow](Development-Workflow.md)
+- [Sections Strucutre](Development-Sections-Structure.md)
```

Comment:
Link doesn't work -- the actual file is just called "Sections-Structure.md"

Reply:
Thank you. Nice catch : )